W-22103685-graphQI-introspection (#1042)
mulesoft/docs-gateway: gateway/1.13/modules/ROOT/pages/policies-included-directory.adoc

| xref:policies-included-dataweave-request-filter.adoc[DataWeave Request Filter] | Transformation | Filters requests by using a DataWeave script
| xref:policies-included-external-authorization.adoc[External Authorization] | Security | Authenticates requests with an external gRPC or HTTP authorization service
| xref:policies-included-external-processing.adoc[External Processing] | Transformation | Sends the incoming HTTP requests or outgoing HTTP responses to an external gRPC service for additional processing

> | xref:policies-included-graphql-introspection-control.adoc[GraphQL Introspection Control] | GraphQL | Lets you allow or block the GraphQL introspection meta-fields `__schema`, `__type`, and `__typename`

Reviewer: ```suggestion
Blocks or allows requests that  access the `__schema`, `__type`, and `__typename` GraphQL meta fields
```

> | xref:policies-included-graphql-introspection-control.adoc[GraphQL Introspection Control] | GraphQL | Lets you allow or block the GraphQL introspection meta-fields `__schema`, `__type`, and `__typename`

Reviewer: ```suggestion
| xref:policies-included-graphql-introspection-control.adoc[GraphQL Introspection Control] | GraphQL | Blocks or allows requests that  access the `__schema`, `__type`, and `__typename` GraphQL meta fields
```
